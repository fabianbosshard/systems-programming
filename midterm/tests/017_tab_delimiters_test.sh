TAB="$(printf '\t')"

exec "$1" 'format=@b @a' delim="${TAB}" <<EOF
Peach${TAB}3${TAB}1
Mango${TAB}4${TAB}2
EOF
