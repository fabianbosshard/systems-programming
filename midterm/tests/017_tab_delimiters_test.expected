3 Peach
4 Mango
